handlers/_swagger.js

```diff
@@ -419,40 +423,52 @@ function validateRequestParameters(req, data, logger, callback) {
 
         } else if (parm.in === 'body') {
             var result = swaggerUtil.validateJSONType(parm.schema, req.body);
-
-            if (!result.valid) {
+            var polymorphicValidationErrors = [];
```

Comment:
Polymorphic validation will catch errors that the previous validation did not, so the idea was to hand back all validation errors